VideoApp/VideoApp/Video/Room/ConnectOptionsFactory.swift

```diff
@@ -35,7 +35,11 @@ import TwilioVideo
                 localVerbosity: .minimal,
                 remoteVerbosity: .minimal
             )
-            
+            builder.bandwidthProfileOptions = BandwidthProfileOptions(
+                videoOptions: VideoBandwidthProfileOptions { builder in
+                    builder.mode = .collaboration
```

Comment:
I did not specify custom render dimensions. I'm a little confused about if I need to do that because documentation kind of implies it's not important. I have more bandwidth profile tickets including one to add settings so we could always optimize it later.